src/Microsoft.TestPlatform.CommunicationUtilities/Messages/MessageType.cs

```diff
@@ -13,31 +20,39 @@ public static class MessageType
     /// <summary>
     /// The session start.
     /// </summary>
+    [ProtocolVersion(Version0, typeof(void), Description = "Not used.")]
```

Comment:
Isn't this the first message sent when initiating a test session (not the test session that pre-starts the testhosts, but the other kind of test session) ? Why does this say it's not used ?

Reply:
The only mention of `MessageType.SessionStart` I could find in our codebase is in public api file. Feel free to correct me.

Reply:
You're right, I was mistaken, just did a search and found nothing either. I guess my memory failed me :D